src/utils/MultiInviter.ts

```diff
@@ -277,6 +277,12 @@ export default class MultiInviter {
                                 errorText = _t("The user's homeserver does not support the version of the room.");
                             }
                             break;
+                        case "ORG.MATRIX.JSSDK_MISSING_PARAM":
```

Comment:
are you sure this is the only reason this error can be returned?

Reply:
Currently yes, especially down the 3pid/email codepaths.